src/Tribe/Admin/Columns/Tickets.php

```diff
@@ -105,11 +105,14 @@ protected function render_tickets_entry( $post_id ) {
 	/**
 	 * Iterates over an array of tickets to fetch the sale total.
 	 *
+	 * @deprecated TBD
+	 *
 	 * @param Tribe__Tickets__Ticket_Object[] $tickets
 	 *
 	 * @return int The sale total.
 	 */
 	protected function get_sold( $tickets ) {
+		_deprecated_function( __METHOD__, 'TBD', 'tribe( "tickets.handler" )->get_ticket_totals()' );
```

Comment:
why are we flagging this as deprecated, but still using it?

Reply:
My bad, Oversight.